app/controllers/not_yet_supported_controller.rb

```diff
@@ -0,0 +1,9 @@
+class NotYetSupportedController < FormsController
+  def self.show_rule_sets(change_report)
+    super << (!change_report.navigator.supported_county? || change_report.is_self_employed_yes?)
```

Comment:
I feel like the way we did this the other day for AddLetterController is a little easier to read:

```
rules = [
  !change_report.navigator.supported_county? ,
  change_report.is_self_employed_yes?,
]
super + rules
```